src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public int getRandomBall() {
```

Comment:
move logic of creating ball using color supplier from Application to here
```suggestion
    public Ball getRandomBall() {
```